projects/vanilla/src/app/app.module.ts

```diff
@@ -73,9 +75,24 @@ import deLocale from "../locales/de";
 import { environment } from "../environments/environment";
 
 // UI Builder
-import { DynamicViewsModule, ConfiguratorModule } from "@sinequa/ngx-ui-builder";
+import {
+    ConditionPipe,
+    ConfiguratorComponent,
+    ConfigModule,
+    ZoneComponent,
+    ToolbarComponent as uibToolbarComponent,
+    AutocompleteComponent as uibAutocompleteComponent,
+    TooltipDirective,
+    ToastComponent,
+    icons,
+    ItemComponent,
+    TemplateNameDirective,
+    SvgIconsModule,
+    NgModelChangeDebouncedDirective,
+    CheckboxComponent,
+    ImageSelectorComponent
+} from '@sinequa/ngx-ui-builder';
```

Comment:
We should have modules to avoid importing all these components/modules one-by-one

Reply:
done